index.js

```diff
@@ -115,6 +115,7 @@ function ParseServer(args) {
   router.merge(require('./installations'));
   router.merge(require('./functions'));
   router.merge(require('./schemas'));
+  router.merge(require('./global_config'));
```

Comment:
ok @drew-gross I've added the experimental flag and updated to `PUT`. I furthermore changed response of it to `result: true` since that's similar to dashboard response.

With regards to testing I setup the config routing in `TESTING=1` mode too .. I guess it makes sense to run tests on experimental features too.